src/languageSupport/languages/flux/lsp/connection.ts

```diff
@@ -329,7 +329,7 @@ export class ConnectionManager {
       // since this._diffSchemaChange() can set the model
       // we need the executeCommand to be issued after the model update
       if (delay) {
-        setTimeout(() => this._updateLsp(toAdd, toRemove), 1500)
+        setTimeout(() => this._updateLsp(toAdd, toRemove), 3000)
```

Comment:
Was getting test flake, if this timeout was <= 2000ms.

Reply:
Same suggestion as in the other PR about giving a name to this [magic number](https://stackoverflow.com/a/47902/92446)

Reply:
Very good idea. Done.